mantidproject/mantid
# Moving 6.4 release notes to manual

**Description of work.**

As part of release for 6.4 the process of dealing with the release notes moves from automatic to manual. This involves copying release notes to what will become the final set of .rst files and moiving the seperate release notes to new Used folders.

Minor tidying of the notes has been done to prevent Sphinx Warnings on the 6.4 release notes. The conf.py file has also been updated to prevent toctree errors for the new Used folders.

**To test:**

- Check that the .rst files in the main 6.4.0 folder (e.g. `diffraction.rst`) now has bullet point lists or blank lines under each heading. The amalgamate directives should no longer be there.
- remove `build/docs/doctrees`
- build the documents again and check there are no warnings for any part of the 6.4.0 release notes (there will be warnings for other files).

*There is no associated issue.*

*This does not require release notes* because **this is the process to move release notes to their final home**.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
